javascript-forms/form-events.js

```diff
@@ -0,0 +1,22 @@
+
+var $userName = document.querySelector('#user-name');
+var $userEmail = document.querySelector('#user-email');
+var $userMessage = document.querySelector('#user-message');
+
+$userName.addEventListener('focus', handleFocus);
+$userEmail.addEventListener('blur', handleBlur);
+$userMessage.addEventListener('input', handleInput);
```

Comment:
You should have an event handler for each of these events on each form control element. Three event listeners on each element.